test-one-license

```diff
@@ -1,9 +1,14 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
+1 to /usr/bin
but -1 to env because of https://jmmv.dev/2016/09/env-considered-harmful.html
Can you just put there /usr/bin/bash ?